Implement navbar for View/Edit Trip Details screen

whiteclouds7 opened the pull request:
closes part of 5 (relates to #5)

Navigation Tab is shown for TripItinerary and Settingsscreen, and TripId is also passed to Settingsscreen (probably not in the nicest way possible, but the documentation is not that great for typescript support, but it works, soooo.... 😉)

whiteclouds7 committed:
update RootStackParamList to use Routes enum

whiteclouds7 committed:
add navbar

whiteclouds7 committed:
Merge remote-tracking branch 'origin/develop' into feature/5-view-edit-trip-details-navbar

# Conflicts:
#	package-lock.json
#	package.json
#	src/screens/Itinerary/TripItinerary.tsx
#	src/types/RootStackParamList.ts

whiteclouds7 committed:
merge with develop

whiteclouds7 committed:
remove console.log

whiteclouds7 committed:
change active color

whiteclouds7 committed:
change active color again

DominicLuidold committed:
Merge branch 'develop' into feature/5-view-edit-trip-details-navbar

DominicLuidold reviewed (approved):
Looks good! Thanks for the implementation @whiteclouds7 👍

whiteclouds7 committed:
moved custom header to TripTabs as it was not shown inside the tab screens

whiteclouds7 committed:
Merge branch 'feature/5-view-edit-trip-details-navbar' of https://github.com/vacatius/mobile-app into feature/5-view-edit-trip-details-navbar

whiteclouds7 commented:
@DominicLuidold can you have a look at it again? I removed the title under the icons, as it used no translations. I also realised that our custom header was not shown inside the Itinerary and Settings screen, but the default one. I've fixed that as well.
Thank you in advance 😊
